pkg/util/sets/BUILD

```diff
@@ -13,14 +13,8 @@ load(
 go_library(
     name = "go_default_library",
     srcs = [
-        "byte.go",
-        "doc.go",
-        "empty.go",
-        "int.go",
-        "int64.go",
-        "string.go",
+        ":set-gen",
```

Comment:
This means use the output of the genrule as the source. :set-gen is the label of the set-gen genrule.